server/src/test/helper.ts

```diff
@@ -58,3 +60,21 @@ export function sleep(ms: number): Promise<unknown> {
     setTimeout(resolve, ms);
   });
 }
+
+export async function shouldProvideCompletions(
+  completionProvider: DbtNodeCompletionProvider,
+  text: string,
+  expectedCompletions: { label: string; insertText: string }[],
```

Comment:
Can we make it the same type as actual completions? `CompletionItem[] | undefined`